shared/admin/js/admin.js

```diff
@@ -309,9 +309,11 @@ var TableHeader = React.createClass({
 
 var TableBody = React.createClass({
     render: function() {
-        var tableRows = this.props.data.values.map(function (row) {
-            return React.createElement(TableRow, {data: row});
```

Comment:
The admin console would get a js error and not display any results if a row came back without any values. Various 'show' queries do this and so I fixed it so I could use them. The fix is to check if the values array is empty and just skip drawing elements if it is.

I believe there is an issue filed for this I'll look it up and add it to the CHANGELOG of this PR.